packages/ui/src/components/Banner/banner.tsx

```diff
@@ -15,6 +18,7 @@ export type BannerProps = {
     imgUrl?: string
     invertText?: boolean
     id?: string
+    mode?: 'compact'
```

Comment:
Not the obvious name for a prop.
Mb `size`?

Reply:
It seems that the changes that occur are not characterized with `size`

This is a different type of banner. So I renamed it to `type`